Gemfile.lock

```diff
@@ -147,17 +149,17 @@ GEM
       json
       thread
       thread_safe
-    elasticsearch (2.0.0)
```

Comment:
This one we want to avoid, the don't use semver. Right now its at elasticsearch 2.0.0; indicating API compatibility with elasticsearch 2.0.0. Upgrading to 5 warrants a corresponding elasticsearch upgrade.

It probably won't break anything too badly in terms of BC, but there are some things to watch out for (or at least that was my experience from 1.4 -> 2.0, a BC break every other release)